examples/vite/.env.example

```
@@ -0,0 +1,3 @@
+INITIA_TEST_MNEMONIC=test test test test test test test test test test test junk
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)